test/verify/machineslib.py

```diff
@@ -196,10 +194,18 @@ def setUp(self):
         # we don't have configuration to open the firewall for local libvirt machines, so just stop firewalld
         m.execute("systemctl stop firewalld; systemctl try-restart libvirtd")
 
-    def tearDown(self):
-        # FIXME: Call `super.tearDown()` or remove this method. There are some unexpected messages
-        # which need to be adjusted before we can do that.
-        pass
+        # FIXME: report downstream; AppArmor noisily denies some operations, but they are not required for us
+        self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="capable" profile="/usr/sbin/libvirtd.* capname="sys_rawio" .*')
+        if m.image in ["ubuntu-2004"]:
+            self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirt.* name="/" .* denied_mask="r" .*')
+            self.allow_journal_messages('.* type=1400 .* apparmor="DENIED" operation="open" profile="libvirt.* name="/sys/bus/nd/devices/" .* denied_mask="r" .*')
+
+        # FIXME: report downstream: qemu often crashes in testAddDisk and testMultipleSettings
+        if m.image in ["ubuntu-stable"]:
+            self.allow_journal_messages('Process .* (qemu-system-x86) of user .* dumped core.')
```

Comment:
This [seems](https://logs.cockpit-project.org/logs/pull-13741-20200317-225449-07ad1ecd-ubuntu-stable/log.html#244-2) not to get applied. My guess are those brackets.